packages/select/src/option-group.vue

```diff
@@ -37,10 +37,11 @@ export default defineComponent({
 
     const select = inject(selectKey)
 
-    const queryChange = () => {
+    const groupQueryChange = toRef(select, 'groupQueryChange')
+
+    watch(groupQueryChange, () => {
       visible.value = select?.options?.some(option => option.visible === true)
```

Comment:
Do we need a default value for this? Though the chaining fallback should be `undefined`

Reply:
But the type of `visible` should be `boolean` instead of `undefined | boolean` right?